libraries/lib-wave-track/WaveClip.cpp

```diff
@@ -1005,7 +1039,8 @@ void WaveClip::Resample(int rate, BasicUI::ProgressDialog *progress)
 // be exactly equal due to rounding errors.
 bool WaveClip::SharesBoundaryWithNextClip(const WaveClip* next) const
 {
-   double endThis = GetRate() * GetPlayStartTime() + GetVisibleSampleCount().as_double();
+   double endThis = GetRate() * GetPlayStartTime() +
+                    GetVisibleSampleCount().as_double() * GetStretchRatio();
```

Comment:
I see this change, and the first thing I think is, was similar stretching of `GetVisibleSampleCount` also done where needed at every other call of that function or use of its underlying member variables?

I don't know the answer just now.  It's not so quick to determine that.  Code review often means examining what did not change, not only what did change.

Reply:
I'm afraid I don't understand.